include/logger/loggerConfig.h

```diff
@@ -260,9 +260,9 @@ typedef struct _PWMConfig{
 } PWMConfig;
 
 /// PWM frequency in Hz.
-#define MAX_PWM_CLOCK_FREQUENCY             200000
-#define MIN_PWM_CLOCK_FREQUENCY	 	10
-#define DEFAULT_PWM_CLOCK_FREQUENCY			10000
+#define MAX_PWM_CLOCK_FREQUENCY             65000
```

Comment:
I noticed a lot of non-kernel style violations as well, but ignored them for the moment.  Is better to go through and address style issues at once rather than ad-hoc to avoid inconsistency IMO.